packages/utils-app/src/App.ts

```diff
@@ -90,15 +103,17 @@ export abstract class App {
         await this.__startMetricsServer(this.options.metricsPort)
       }
 
+      // Start admin api server if enabled
+      if (this.options.adminEnabled) {
+        await this.__startAdminApiServer(this.options.adminPort)
+      }
+
       // Run pre-main hook
       await this.preMain()
 
       // Run main hook
       await this.main()
 
-      // Run post-main hook
-      await this.postMain()
-
```

Comment:
was this intentionally removed?

Reply:
ah yea, i figured it overlaps with `shutdown` and wont be used